package.json

```diff
@@ -5,8 +5,6 @@
         "dependencies": {
                 "@augloop/types-core": "file:packages/types-core-2.16.189.tgz",
                 "@axe-core/webdriverjs": "4.10.0",
-                "@fluentui/react-components": "9.55.1",
-                "@fluentui/react-icons": "2.0.264",
```

Comment:
Oh we missed this, let's not remove the packages

Reply:
they had been repeated